x/evm/genesis_test.go

```diff
@@ -45,9 +44,7 @@ var _ = g.Describe("genesis", func() {
 	g.BeforeEach(func() {
 		t := g.GinkgoT()
 		a = app.NewTestApp(t, false)
-		ctx = a.NewContext(false, tmproto.Header{
-			Height: 5,
-		})
```

Comment:
I think the height had a reason for being in the test.

Reply:
height is also included in [ctx](url)